coralproject/talk
# do not highlight partial word matches in admin since this is not how the backend works

## What does this PR do?

if you had a naughty word in either the suspect or banned list, it was handled correctly on the backend, but it was highlighted in the queue improperly. if you had banned "butt", the first syllable of "buttress" would be highlighted. This gave the impression that we hadn't properly programmed the word service. Now it's fixed!

## How do I test this PR?

- add some short words to the banned/suspect word lists
- make a few comments
  - one where the bad word match would be at the very beginning of the string e.g. `butt I wanted to go to the store`
  - one where the bad word match is somewhere in the middle e.g `I have to use the word bottom instead of butt with my toddler`
  - one where it's at the very end `anything butt`
- partial words should not be highlighted, only words with spaces on either side or at the beginning/end of the comment string.

## known issues
I have to figure out how to highlight the word even if it's wrapped in quotes.
